src/public/V8JsiRuntime.h

```diff
@@ -76,13 +76,11 @@ struct V8RuntimeArgs {
 
   bool backgroundMode{false};
 
-  // Enabling all the diagnostic and tracings by default so that we will be able to tune them and writing tools over them.
-  bool trackGCObjectStats{true};
-  bool enableTracing{true};
-  bool enableJitTracing{true};
-  bool enableMessageTracing{true};
-  bool enableLog{true};

```

Comment:
>bool enableLog{true}; [](start = 2, length = 21)

Can you keep enableLog in there even if not used (maybe with a comment)? Otherwise, the change will break the ABI meaning we'd need to recompile all dependencies up-the-chain from source instead of just dropping a new DLL directly in the consumer.